Grab_that_article_on_Medium.py

```diff
@@ -104,19 +104,20 @@
 links = [i.get_attribute('href') for i in driver.find_elements_by_xpath("//a[@data-action-source]")]
 unique_list = list(OrderedDict(zip(links, repeat(None)))) # To remove duplicates from list
 
-f= open("output.txt","a+") # Stores ouput in output.txt in the same file directory
+f= open("output.txt","a+",encoding='utf8') # Stores ouput in output.txt in the same file directory
 if topic==0:               # Different structure of Home and other topic pages.
  if tag_len > 0:
     for i,l in zip(range(tag_len), unique_list):
         f.write(time.strftime("%Y-%m-%d %H:%M") + '\n')
         f.write(tag[i].text)
         f.write('\nLink is -->   ' + str(l)  + '\n\n')
 else:
-    for i in range(tag_len):
-        f.write(time.strftime("%Y-%m-%d %H:%M") + '\n')
-        f.write(tag[i].text)
-        l = tag[i].find_element_by_css_selector('a').get_attribute('href');
-        f.write('\nLink is -->   ' + str(l)  + '\n\n')
+    for i in range(52):
```

Comment:
Why 52?

Reply:
Line 116 - `if i!=0 and i!=4:                                #Eliminating the blank 1st entry and 'Featured Writer'`
This line eliminates 2 entries from the list. Thus to maintain 50 entries in the list I wrote 52.
For i=0 the `tag[0].text` is null.
And for i=4 it gives a 'Featured Writer' link which is not actually an article.